### PR TITLE
Admin form

### PR DESCRIPTION
**Pull request by podarok**
image

Options setters not working yet. 
Config names should be rethinked, I guess. Or submit handler should be improved

### CODE REVIEW

**Comment by podarok** [comment]
@Sanchiz review, please

File: d8/modules/custom/advanced_forum/advanced_forum.routing.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+advanced_forum.admin_settings:
+  path: '/admin/config/content/advanced-forum'
+  defaults:
+    _form: '\Drupal\advanced_forum\Form\AdminSettingsForm'
+    _title: 'Advanced Forum settings'
+  requirements:
+    _permission: 'administer advanced forum'
```
Comment (alexschedrov) [inline d8/modules/custom/advanced_forum/advanced_forum.routing.yml]: Endline

File: d8/modules/custom/advanced_forum/src/Form/AdminSettingsForm.php [hunk 2]
```diff
@@ -0,0 +1,365 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: podarok
+ * Date: 30.12.15
+ * Time: 13:07
```
Comment (alexschedrov) [inline d8/modules/custom/advanced_forum/src/Form/AdminSettingsForm.php]: Do we really need this metadata?

File: d8/modules/custom/advanced_forum/src/Form/AdminSettingsForm.php [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+      '#title' => t('Containers collapsed by default'),
+      // @d7 '#default_value' => variable_get('advanced_forum_default_collapsed_list', array()),
+      '#default_value' => $this->config('advanced_forum_lists')->get('advanced_forum_default_collapsed_list'),
+      '#options' => $collapsed_list_description,
+      '#multiple' => TRUE,
+      '#description' => t('Select containers which should be collapsed by default.'),
+    );
+
+    if (\Drupal::moduleHandler()->moduleExists('image')) {
+
```
Comment (alexschedrov) [inline d8/modules/custom/advanced_forum/src/Form/AdminSettingsForm.php:211]: Empty line.

File: d8/modules/custom/advanced_forum/src/Form/AdminSettingsForm.php [hunk 4]
```diff
@@ -0,0 +1,365 @@
...
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    // @todo implement submit handler.
+  }
+
+}
```
Comment (Sanchiz) [inline d8/modules/custom/advanced_forum/src/Form/AdminSettingsForm.php]: New line

**Pull request merged**